### PR TITLE
Aggregation of cosmetic changes made during work on REPL PRs: misc.

### PR DESCRIPTION
**Pull request by alexreg**
Factored out from hacking on rustc for work on the REPL.

r? @Centril

### CODE REVIEW

File: src/librustc_codegen_ssa/traits/builder.rs [hunk 1]
```diff
@@ -256,10 +256,10 @@ pub trait BuilderMethods<'a, 'tcx>:
     fn atomic_fence(&mut self, order: AtomicOrdering, scope: SynchronizationScope);
     fn set_invariant_load(&mut self, load: Self::Value);
 
-    /// Called for `StorageLive`
```
Comment (Centril) [inline src/librustc_codegen_ssa/traits/builder.rs]: Revert removal of backticks.

File: src/librustc_data_structures/fingerprint.rs [hunk 2]
```diff
@@ -85,16 +85,17 @@ impl stable_hasher::StableHasherResult for Fingerprint {
 impl_stable_hash_via_hash!(Fingerprint);
 
 impl rustc_serialize::UseSpecializedEncodable for Fingerprint { }
-
 impl rustc_serialize::UseSpecializedDecodable for Fingerprint { }
 
 impl rustc_serialize::SpecializedEncoder<Fingerprint> for Encoder {
+    #[inline]
```
Comment (Centril) [inline src/librustc_data_structures/fingerprint.rs]: Semantic change.

File: src/librustc_data_structures/fingerprint.rs [hunk 3]
```diff
@@ -85,16 +85,17 @@ impl stable_hasher::StableHasherResult for Fingerprint {
...
     fn specialized_encode(&mut self, f: &Fingerprint) -> Result<(), Self::Error> {
         f.encode_opaque(self)
     }
 }
 
 impl<'a> rustc_serialize::SpecializedDecoder<Fingerprint> for Decoder<'a> {
+    #[inline]
```
Comment (Centril) [inline src/librustc_data_structures/fingerprint.rs]: Semantic change.

File: src/librustc_interface/interface.rs [hunk 4]
```diff
@@ -21,13 +21,13 @@ use syntax_pos::edition;
...
 
-/// Represents a compiler session.
+/// Represents a compiler instance.
 /// Can be used run `rustc_interface` queries.
 /// Created by passing `Config` to `run_compiler`.
 pub struct Compiler {
     pub(crate) sess: Lrc<Session>,
-    codegen_backend: Lrc<Box<dyn CodegenBackend>>,
-    source_map: Lrc<SourceMap>,
+    pub codegen_backend: Lrc<Box<dyn CodegenBackend>>,
```
Comment (Centril) [inline src/librustc_interface/interface.rs]: Semantic changes here.

File: src/librustc_interface/queries.rs [hunk 5]
```diff
@@ -71,7 +71,7 @@ impl<T> Default for Query<T> {
 }
 
 #[derive(Default)]
-pub(crate) struct Queries {
+pub struct Queries {
```
Comment (Centril) [inline src/librustc_interface/queries.rs]: Semantic change here.

File: src/librustc_metadata/decoder.rs [hunk 6]
```diff
@@ -362,6 +362,7 @@ impl SpecializedDecoder<Ident> for DecodeContext<'_, '_> {
 }
 
 impl<'a, 'tcx> SpecializedDecoder<Fingerprint> for DecodeContext<'a, 'tcx> {
+    #[inline]
```
Comment (Centril) [inline src/librustc_metadata/decoder.rs]: Semantic change here.

File: src/librustc_metadata/encoder.rs [hunk 7]
```diff
@@ -213,6 +215,7 @@ impl<'tcx> SpecializedEncoder<interpret::AllocId> for EncodeContext<'tcx> {
 }
 
 impl<'tcx> SpecializedEncoder<ty::GenericPredicates<'tcx>> for EncodeContext<'tcx> {
+    #[inline]
```
Comment (Centril) [inline src/librustc_metadata/encoder.rs]: Semantic change here.

File: src/librustc_metadata/encoder.rs [hunk 8]
```diff
@@ -221,12 +224,14 @@ impl<'tcx> SpecializedEncoder<ty::GenericPredicates<'tcx>> for EncodeContext<'tc
 }
 
 impl<'tcx> SpecializedEncoder<Fingerprint> for EncodeContext<'tcx> {
+    #[inline]
```
Comment (Centril) [inline src/librustc_metadata/encoder.rs]: Semantic change here.

File: src/librustc_metadata/encoder.rs [hunk 9]
```diff
@@ -221,12 +224,14 @@ impl<'tcx> SpecializedEncoder<ty::GenericPredicates<'tcx>> for EncodeContext<'tc
...
     fn specialized_encode(&mut self, f: &Fingerprint) -> Result<(), Self::Error> {
         f.encode_opaque(&mut self.opaque)
     }
 }
 
 impl<'tcx, T: Encodable> SpecializedEncoder<mir::ClearCrossCrate<T>> for EncodeContext<'tcx> {
+    #[inline]
```
Comment (Centril) [inline src/librustc_metadata/encoder.rs]: Semantic change here.

File: src/librustc_typeck/astconv.rs [hunk 10]
```diff
@@ -1,4 +1,4 @@
-//! Conversion from AST representation of types to the `ty.rs` representation.
+//! Conversion from AST representation of types to the typesystem representation.
```
Comment (Centril) [inline src/librustc_typeck/astconv.rs]:
```suggestion
//! Conversion from AST representation of types to the type-system representation.
```

File: src/librustc_typeck/astconv.rs [hunk 11]
```diff
@@ -1212,17 +1208,16 @@ impl<'o, 'tcx> dyn AstConv<'tcx> + 'o {
     fn trait_ref_to_existential(&self, trait_ref: ty::TraitRef<'tcx>)
                                 -> ty::ExistentialTraitRef<'tcx> {
         if trait_ref.self_ty() != self.tcx().types.trait_object_dummy_self {
-            bug!("trait_ref_to_existential called on {:?} with non-dummy Self", trait_ref);
+            bug!("trait_ref_to_existential called on {:?} with non-dummy self type", trait_ref);
```
Comment (Centril) [inline src/librustc_typeck/astconv.rs]:
```suggestion
            bug!("trait_ref_to_existential called on {:?} with non-dummy Self type", trait_ref);
```

File: src/libsyntax_ext/deriving/decodable.rs [hunk 12]
```diff
@@ -18,42 +19,54 @@ pub fn expand_deriving_rustc_decodable(cx: &mut ExtCtxt<'_>,
...
     let typaram = "__D";
+    let decodable_path = Path::new_(vec![krate, "Decodable"], None, vec![], PathKind::Global);
 
     let trait_def = TraitDef {
         span,
         attributes: Vec::new(),
-        path: Path::new_(vec![krate, "Decodable"], None, vec![], PathKind::Global),
-        additional_bounds: Vec::new(),
+        path: decodable_path.clone(),
+        additional_bounds: vec![Ty::Literal(decodable_path)],
```
Comment (Centril) [inline src/libsyntax_ext/deriving/decodable.rs]: Semantic change here.

File: src/libsyntax_ext/deriving/encodable.rs [hunk 13]
```diff
@@ -102,12 +103,13 @@ pub fn expand_deriving_rustc_encodable(cx: &mut ExtCtxt<'_>,
...
     let typaram = "__S";
+    let encodable_path = Path::new_(vec![krate, "Encodable"], None, vec![], PathKind::Global);
 
     let trait_def = TraitDef {
         span,
         attributes: Vec::new(),
-        path: Path::new_(vec![krate, "Encodable"], None, vec![], PathKind::Global),
-        additional_bounds: Vec::new(),
+        path: encodable_path.clone(),
+        additional_bounds: vec![Ty::Literal(encodable_path)],
```
Comment (Centril) [inline src/libsyntax_ext/deriving/encodable.rs]: Semantic change here.

File: src/libsyntax_ext/deriving/generic/mod.rs [hunk 14]
```diff
@@ -201,25 +201,25 @@ use crate::deriving;
...
 
 pub struct TraitDef<'a> {
-    /// The span for the current #[derive(Foo)] header.
+    /// The span for the current `#[derive(Foo)]` header.
     pub span: Span,
 
     pub attributes: Vec<ast::Attribute>,
 
-    /// Path of the trait, including any type parameters
+    /// The path of the trait, including any type parameters
```
Comment (Centril) [inline src/libsyntax_ext/deriving/generic/mod.rs]:
```suggestion
    /// The path of the trait, including any type parameters.
```

File: src/libsyntax_ext/deriving/generic/mod.rs [hunk 15]
```diff
@@ -201,25 +201,25 @@ use crate::deriving;
...
-    /// Any extra lifetimes and/or bounds, e.g., `D: serialize::Decoder`
+    /// Any extra lifetimes and/or bounds, e.g., `D: serialize::Decoder`.
     pub generics: LifetimeBounds<'a>,
 
-    /// Is it an `unsafe` trait?
+    /// `true` if it is an `unsafe` trait.
     pub is_unsafe: bool,
 
-    /// Can this trait be derived for unions?
+    /// `true` if this trait be derived for unions.
```
Comment (Centril) [inline src/libsyntax_ext/deriving/generic/mod.rs]:
```suggestion
    /// `true` if this trait be derived for `union`s.
```

File: src/libsyntax_ext/deriving/generic/mod.rs [hunk 16]
```diff
@@ -1226,7 +1223,7 @@ impl<'a> MethodDef<'a> {
         // (Variant1, Variant1, ...) => Body1
         // (Variant2, Variant2, ...) => Body2
         // ...
-        // where each tuple has length = self_args.len()
+        // where each tuple has length of `self_args.len()`.
```
Comment (Centril) [inline src/libsyntax_ext/deriving/generic/mod.rs]:
```suggestion
        // where each tuple has a length of `self_args.len()`.
```

File: src/libsyntax_ext/deriving/generic/mod.rs [hunk 17]
```diff
@@ -1646,13 +1642,13 @@ impl<'a> TraitDef<'a> {
          -> (P<ast::Pat>, Vec<(Span, Option<Ident>, P<Expr>, &'a [ast::Attribute])>) {
         let sp = variant.span.with_ctxt(self.span.ctxt());
         let variant_path = cx.path(sp, vec![enum_ident, variant.ident]);
-        let use_temporaries = false; // enums can't be repr(packed)
+        let use_temporaries = false; // Enums can't be `repr(packed)`.
```
Comment (Centril) [inline src/libsyntax_ext/deriving/generic/mod.rs]:
```suggestion
        let use_temporaries = false; // `enum`s cannot be `repr(packed)`.
```

File: src/libsyntax_pos/lib.rs [hunk 18]
```diff
@@ -929,9 +929,9 @@ impl Encodable for SourceFile {
             s.emit_struct_field("name", 0, |s| self.name.encode(s))?;
             s.emit_struct_field("name_was_remapped", 1, |s| self.name_was_remapped.encode(s))?;
             s.emit_struct_field("src_hash", 2, |s| self.src_hash.encode(s))?;
-            s.emit_struct_field("start_pos", 4, |s| self.start_pos.encode(s))?;
-            s.emit_struct_field("end_pos", 5, |s| self.end_pos.encode(s))?;
-            s.emit_struct_field("lines", 6, |s| {
+            s.emit_struct_field("start_pos", 3, |s| self.start_pos.encode(s))?;
```
Comment (Centril) [inline src/libsyntax_pos/lib.rs]: Semantic changes here, revert.

File: src/libsyntax_pos/lib.rs [hunk 19]
```diff
@@ -977,13 +977,13 @@ impl Encodable for SourceFile {
 
                 Ok(())
             })?;
-            s.emit_struct_field("multibyte_chars", 7, |s| {
+            s.emit_struct_field("multibyte_chars", 6, |s| {
```
Comment (Centril) [inline src/libsyntax_pos/lib.rs]: Semantic changes here, revert.

File: src/libsyntax_pos/lib.rs [hunk 20]
```diff
@@ -977,13 +977,13 @@ impl Encodable for SourceFile {
...
                 self.multibyte_chars.encode(s)
             })?;
-            s.emit_struct_field("non_narrow_chars", 8, |s| {
+            s.emit_struct_field("non_narrow_chars", 7, |s| {
```
Comment (Centril) [inline src/libsyntax_pos/lib.rs]: Semantic changes here, revert.

File: src/libsyntax_pos/lib.rs [hunk 21]
```diff
@@ -977,13 +977,13 @@ impl Encodable for SourceFile {
...
                 self.non_narrow_chars.encode(s)
             })?;
-            s.emit_struct_field("name_hash", 9, |s| {
+            s.emit_struct_field("name_hash", 8, |s| {
```
Comment (Centril) [inline src/libsyntax_pos/lib.rs]: Semantic changes here, revert.

File: src/libsyntax_pos/lib.rs [hunk 22]
```diff
@@ -992,17 +992,16 @@ impl Encodable for SourceFile {
...
             let name: FileName = d.read_struct_field("name", 0, |d| Decodable::decode(d))?;
             let name_was_remapped: bool =
                 d.read_struct_field("name_was_remapped", 1, |d| Decodable::decode(d))?;
             let src_hash: u128 =
                 d.read_struct_field("src_hash", 2, |d| Decodable::decode(d))?;
             let start_pos: BytePos =
-                d.read_struct_field("start_pos", 4, |d| Decodable::decode(d))?;
-            let end_pos: BytePos = d.read_struct_field("end_pos", 5, |d| Decodable::decode(d))?;
-            let lines: Vec<BytePos> = d.read_struct_field("lines", 6, |d| {
+                d.read_struct_field("start_pos", 3, |d| Decodable::decode(d))?;
```
Comment (Centril) [inline src/libsyntax_pos/lib.rs]: Semantic changes here, revert.

File: src/libsyntax_pos/lib.rs [hunk 23]
```diff
@@ -1031,18 +1030,18 @@ impl Decodable for SourceFile {
                 Ok(lines)
             })?;
             let multibyte_chars: Vec<MultiByteChar> =
-                d.read_struct_field("multibyte_chars", 7, |d| Decodable::decode(d))?;
+                d.read_struct_field("multibyte_chars", 6, |d| Decodable::decode(d))?;
```
Comment (Centril) [inline src/libsyntax_pos/lib.rs]: Semantic changes here, revert.

File: src/libsyntax_pos/lib.rs [hunk 24]
```diff
@@ -1031,18 +1030,18 @@ impl Decodable for SourceFile {
...
             let non_narrow_chars: Vec<NonNarrowChar> =
-                d.read_struct_field("non_narrow_chars", 8, |d| Decodable::decode(d))?;
+                d.read_struct_field("non_narrow_chars", 7, |d| Decodable::decode(d))?;
```
Comment (Centril) [inline src/libsyntax_pos/lib.rs]: Semantic changes here, revert.

File: src/libsyntax_pos/lib.rs [hunk 25]
```diff
@@ -1031,18 +1030,18 @@ impl Decodable for SourceFile {
...
             let name_hash: u128 =
-                d.read_struct_field("name_hash", 9, |d| Decodable::decode(d))?;
+                d.read_struct_field("name_hash", 8, |d| Decodable::decode(d))?;
```
Comment (Centril) [inline src/libsyntax_pos/lib.rs]: Semantic changes here, revert.

File: src/test/mir-opt/inline-retag.rs [hunk 26]
```diff
@@ -1,6 +1,6 @@
 // compile-flags: -Z span_free_formats -Z mir-emit-retag
 
-// Tests that MIR inliner fixes up `Retag`'s `fn_entry` flag
+// Tests that MIR inliner fixes up Retag's `fn_entry` flag.
```
Comment (Centril) [inline src/test/mir-opt/inline-retag.rs]: revert; should have backticks.

File: src/test/ui/nll/polonius/storagedead-kills-loans.rs [hunk 27]
```diff
@@ -1,4 +1,4 @@
-// Whenever a `StorageDead` MIR statement destroys a value `x`,
+// Whenever a StorageDead MIR statement destroys a value `x`,
```
Comment (Centril) [inline src/test/ui/nll/polonius/storagedead-kills-loans.rs]: revert, should have backticks.

**Comment by oli-obk** [comment]
Thanks for the PR. Unfortunately, as per our rules set up in #58619, I am closing this PR.

**Pull request closed without merging**